docs/framework/tools/mage-exe-manifest-generation-and-editing-tool.md

```diff
@@ -73,11 +73,13 @@ The following table shows the options supported by the `-Sign` command, which ap
 
 | Options | Description |
 |---------|-------------|
+|**-a, -Algorithm**|Specifies the algorithm to generate dependency digests with. Value must be "sha256RSA" or "sha1RSA".|
```

Comment:
```suggestion
|**-a, -Algorithm**|Specifies the algorithm to generate dependency digests with. The value must be "sha256RSA".|
```

Reply:
Ah, missed that detail when implementing changes - thanks for catching this!